composer.json

```diff
@@ -9,7 +9,14 @@
     "ext-dom": "*",
     "ext-json": "*",
     "ext-libxml": "*",
-    "ext-mbstring": "*"
+    "ext-mbstring": "*",
+    "ampproject/amp-wp": "^2.0.2",
+    "cweagans/composer-patches": "^1.6",
+    "mcaskill/composer-exclude-files": "^1.2"
```

Comment:
```suggestion
    "mcaskill/composer-exclude-files": "^2.0"
```

Reply:
Thank you for chiming in and providing the fixes!